time received_time
uint32 last_command
uint32 split_number
uint32 current_split_index
communication_software/Message[] data
